mrobosub_localization/src/localization.py

```diff
@@ -39,12 +41,15 @@ class StateEstimation(Node):
     orientation = None
 
     def __init__(self):
+        self.iekf_class = IEKF()
         super().__init__('localization')
         self.heave_pub = rospy.Publisher('/pose/heave', Float64, queue_size=1)
         self.yaw_pub = rospy.Publisher('/pose/yaw', Float64, queue_size=1)
         self.pitch_pub = rospy.Publisher('/pose/pitch', Float64, queue_size=1)
-        self.roll_pub = rospy.Publisher('/pose/roll', Float64, queue_size=1)   
-        rospy.Subscriber('/iekf/state', Float64, self.iekf_callback)
+        self.roll_pub = rospy.Publisher('/pose/roll', Float64, queue_size=1) 
+        self.x_pub = rospy.Publisher('/pose/x_pos', Float64, queue_size=1)
```

Comment:
Could you just add a comment here saying that this is x_pos and y_pos in the global frame (which is why its not surge and sway)